src/components/NewsArticle/NewsArticle.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { Article } from "../../types/article";
+import styles from "./NewsArticle.module.css";
+
+interface ArticleProps {
+  article: Article;
+  key: string;
+}
+
+export const NewsArticle = ({ article, key }: ArticleProps) => {
+  return (
+    <div className={styles.articleBox} key={key}>
+      <div style={{ height: "80px" }}>
+        <a href={article.url} className={styles.articleTitle}>
+          {article.title}
+        </a>
+      </div>
+      {article.publicationDate && (
+        <div className={styles.publicationDateBox}>
+          <p className={styles.publicationDateText}>
```

Comment:
add a space for description, it can be blank for now since none of the articles have that but we want to be ready to support